favicon/README.md

```diff
@@ -0,0 +1,54 @@
+# Purescript Favicon
+
+Besides the well-known `favicon.ico` that is shown in browsers to identify the website, there are several other images we can make available to browsers for use in home screen icons, tiles, etc.
+
+## Usage
+
+Insert the following code in the <head> section of your pages:
+
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="/manifest.json">
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#000000">
+    <link rel="shortcut icon" href="/favicon.ico">
+    <meta name="apple-mobile-web-app-title" content="Pursuit">
+    <meta name="application-name" content="Pursuit">
+    <meta name="msapplication-config" content="/browserconfig.xml">
+    <meta name="theme-color" content="#000000">
+
+## Files used
+
+* `favicon.ico` – Used by IE, and also by some other browsers if we are not careful.
+* `favicon-16x16.png` – The classic favicon, displayed in the tabs.
+* `favicon-32x32.png` – For Safari on Mac OS.
+* `android-chrome-192x192.png` – For Android Chrome M39+
+* `android-chrome-512x512.png` – For Android Chrome M39+
+* `mstile-70x70.png` – For Windows 8 / IE11.
+* `mstile-150x150.png` – For Windows 8 / IE11.
+* `mstile-310x310.png` – For Windows 8 / IE11.
+* `mstile-310x150.png` – For Windows 8 / IE11.
+* `apple-touch-icon.png` – For iOS home screens.
+
+These images were generated by [Real Favicon Generator](https://realfavicongenerator.net) on 2017-01-07. The following source files taken from the [Purescript Logo](https://github.com/purescript/logo) repository were used:
```

Comment:
Probably don't need to specify where the source files came from since they are in the same directory.